### DISCUSSION TITLE
How to simply play a mp3 bgm on the laptop which is running Jamulus and send the bgm to another peer of Jamulus

### DISCUSSION THREAD

**Discussion by Dizzy1976** [jamulussoftware/jamulus#2003] [General]
I want to jam with my friend. he plays guitar, but I'm not going to play any instrument on my side. I just want to play a bgm on my computer which is running Jamulus. I expect my friend can hear that through Jamulus just like I'm playing that bgm.

How can I make it with easy way?

**Comment by ghost** [comment]
The bgm will have to come from an external bgm device because Jamulus and sound Apps demand conflicting sound resources.
